x-pack/platform/plugins/private/painless_lab/ui_tests/fixtures/page_objects/painless_lab_page.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Locator } from '@kbn/scout';
+import { ScoutPage } from '@kbn/scout/src/playwright';
+
+export class PainlessLab {
+  public outputValueElement: Locator;
+  public requestFlyoutHeader: Locator;
+  public viewRequestButton: Locator;
+  public flyoutResponseTab: Locator;
+
+  constructor(private readonly page: ScoutPage) {
+    this.outputValueElement = this.page.testSubj.locator('painlessTabs-loaded');
```

Comment:
I would call it either `editorRightPane` since its class in DOM is called `painlessLabRightPane`